Common/src/main/java/com/tabnineCommon/chat/commandHandlers/context/DiagnosticsContext.kt

```diff
@@ -59,5 +66,14 @@ data class DiagnosticsContext(
                 null
             }
         }
+
+        private fun getSelectedRange(editor: Editor): TextRange? {
```

Comment:
just make sure it actually works, im having PTSD from JB's read/write permissions...